tests/FSharp.Build.UnitTests/WriteCodeFragmentTests.fs

```diff
@@ -96,4 +164,38 @@ type WriteCodeFragmentVisualBasicTests() =
     member _.``Escaped string parameters``() =
         verifyAttribute "SomeAttribute" [("_Parameter1", "\"uno\"")] "SomeAttribute(\"\\\"uno\\\"\")"
         //                                     this should look like: SomeAttribute("\"uno\"")
+        
+    [<Fact>]
+    member _.``Named parameters with _IsLiteral suffix``() =
+        verifyAttribute "SomeAttribute" [("Bool", "true"); ("Bool_IsLiteral", "true")] "SomeAttribute(Bool = true)"
+        
+    [<Fact>]
+    member _.``Multiple named parameters with _IsLiteral suffix``() =
+        verifyAttribute "SomeAttribute" 
+            [
+                ("Number", "42"); 
+                ("Number_IsLiteral", "true"); 
+                ("Bool", "false"); 
+                ("Bool_IsLiteral", "true")
+            ] 
+            "SomeAttribute(Number = 42, Bool = false)"
+            
+    [<Fact>]
+    member _.``Mixed named parameters with and without _IsLiteral suffix``() =
+        verifyAttribute "SomeAttribute" 
+            [
+                ("Number", "42"); 
+                ("Number_IsLiteral", "true"); 
+                ("Text", "hello")
+            ] 
+            "SomeAttribute(Number = 42, Text = \"hello\")"
+            
+    [<Fact>]
+    member _.``Enum _IsLiteral suffix``() =
+        verifyAttribute "SomeAttribute" 
+            [
+                ("EnumValue", "System.StringComparison.OrdinalIgnoreCase"); 
```

Comment:
I did not find any docs for it, but I believe custom msbuild metadata is an alphabetically ordered dictionairy when it goes to the task. Meaning, the original order provided when declaring the properties is not what it ends up being as metadata - those are order by keys.

Long story short, try changing all your assertions; i.e. the final ` "SomeAttribute(...` expected values, to have:
- unnamed properties first. Those come as declared Parameter1,Parameter2, etc.
- named properties ordered alphabetically

Remember, I am writing you to change the expected attribute string. The declaration by the user can be in any order and can mix up unnamed ( `_Parameter1`) and named parameters just fine.